inc/options/objectcache.php

```diff
@@ -121,24 +121,52 @@
 		<table class="form-table">
 			<?php
 			$c           = Dispatcher::config();
+			$disabled    = ! $c->get_boolean( 'objectcache.enabled' );
 			$wp_disabled = ! $c->get_boolean( 'objectcache.wp_cron' );
 
+			if ( $disabled ) {
+				echo wp_kses(
+					sprintf(
+						// Translators: 1 opening HTML div tag followed by opening HTML p tag, 2 opening HTML a tag,
+						// Translators: 3 closing HTML a tag, 4 closing HTML p tag followed by closing HTML div tag.
+						__( '%1$sObject Cache is disabled! Enable it %2$shere%3$s to enable this feature.%4$s', 'w3-total-cache' ),
+						'<div class="notice notice-error inline"><p>',
+						'<a href="' . esc_url( admin_url( 'admin.php?page=w3tc_general#object_cache' ) ) . '">',
+						'</a>',
+						'</p></div>'
+					),
+					array(
+						'div' => array(
+							'class' => array(),
+						),
+						'p'   => array(),
+						'a'   => array(
+							'href' => array(),
+						),
+					)
+				);
+			}
+
 			Util_Ui::config_item(
 				array(
 					'key'            => 'objectcache.wp_cron',
 					'label'          => esc_html__( 'Enable WP-Cron Event', 'w3-total-cache' ),
 					'checkbox_label' => esc_html__( 'Enable', 'w3-total-cache' ),
 					'control'        => 'checkbox',
 					'description'    => esc_html__( 'Enabling this will schedule a WP-Cron event that will flush the Object Cache. If you prefer to use a system cron job instead of WP-Cron, you can schedule the following command to run at your desired interval: "wp w3tc flush object".', 'w3-total-cache' ),
+					'disabled'       => $disabled,
 				)
 			);
 
 			$time_options = array();
+			$timezone     = new \DateTimeZone( get_user_meta( get_current_user_id(), 'timezone', true ) ?: wp_timezone()->getName() );
```

Comment:
Users don't have an option to choose a timezone different from the WP site timezone, so there is no "timezone" in user meta.  You can just set the variable to `wp_timezone()`.  If it's only used once below, just use `wp_timezone()` there.

Reply:
Updated